videobuffer.c

```diff
@@ -682,6 +683,11 @@ time_t cVideoBufferRecording::GetRefTime()
   return m_Recording->Start();
 }
 
+void cVideoBufferRecording::AttachInput(bool attach)
```

Comment:
where ist this called? rcordingsBuffer has no videoInput from where this can be called.

Reply:
cLiveStreamer::Action() retunes and opens a videoInput calling it.

Please see and fix yourself:

```
Oct 12 14:43:38 htpc vdr[847]: [960] VNSI: Client with ID 4 connected: 127.0.0.1:41874
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI Client 4->127.0.0.1:41874 thread started (pid=847, tid=1924, prio=high)
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: Welcome client 'XBMC Media Center' with protocol version '10'
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: LiveStreamer::Close - close
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: close video input ...
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: Open recording: /storage/recordings/Rote_Rosen/Rote_Rosen_(2523)/2017-10-12.14.07.1-0.rec
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: File 0 found, size: 2097423128, totalLength now 2097423128
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: File 1 found, size: 1284738608, totalLength now 3382161736
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: total frames: 109914
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: Successfully switched to channel 1 - Das Erste HD
Oct 12 14:43:38 htpc vdr[847]: [1924] VNSI: Started streaming of channel Das Erste HD (timeout 10 seconds)
Oct 12 14:43:38 htpc vdr[847]: [1925] cLiveStreamer stream processor thread started (pid=847, tid=1925, prio=high)
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: re-tuning...
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: close video input ...
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: Successfully found following device: 0x212b340 (1) for receiving, priority=0
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: Dummy receiver (0x7f0764000dc0) activated
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: activate live receiver: 1
Oct 12 14:43:38 htpc vdr[847]: [1925] VNSI: openFile called for index 0 string:/storage/recordings/Rote_Rosen/Rote_Rosen_(2523)/2017-10-12.14.07.1-0.rec/00001.ts
```